lib/shelley/test/unit/Cardano/Wallet/Shelley/Launch/BlockfrostSpec.hs

```diff
@@ -0,0 +1,41 @@
+module Cardano.Wallet.Shelley.Launch.BlockfrostSpec
+    ( spec
+    ) where
+
+import Prelude
+
+import qualified Blockfrost.Client.Types as Blockfrost
+import qualified Data.Text as T
+
+import Blockfrost.Env
+    ( Env (Testnet) )
+import Cardano.Wallet.Shelley.Launch.Blockfrost
+    ( readToken, tokenFileOption )
+import Options.Applicative
+    ( ParserResult (CompletionInvoked, Failure, Success)
+    , defaultPrefs
+    , execParserPure
+    , fullDesc
+    , info
+    )
+import Test.Hspec
+    ( Spec, describe, expectationFailure, it, shouldReturn )
+import UnliftIO
+    ( withSystemTempFile )
+import UnliftIO.IO
+    ( hClose )
+
+spec :: Spec
+spec = describe "Blockfrost CLI options" $
+    it "tokenFileOption" $ withSystemTempFile "blockfrost.token" $ \fp h -> do
+        let parserInfo = info tokenFileOption fullDesc
+            args = ["--blockfrost-token-file", fp]
+            net = "testnet"
+            projectId = "jlUej4vcMt3nKPRAiNpLUEeKBIEPqgH2"
```

Comment:
Just for completeness: This is just a dummy token, not an actual secret, right?

Reply:
sure thing :)